jni/src/nmslib_wrapper.cpp

```diff
@@ -27,301 +27,354 @@
 #include <string>
 
 #include "hnswquery.h"
+#include "method/hnsw.h"
 
```

Comment:
Please search 'LoadIndexWithStream' in `nmslib_wrapper.cpp` and ignore the formatting changes. :melting_face: